src/java/org/apache/cassandra/auth/AuthConfig.java

```diff
@@ -82,7 +85,7 @@ public static void applyAuth()
         IRoleManager roleManager = authInstantiate(conf.role_manager, CassandraRoleManager.class);
 
         if (authenticator instanceof PasswordAuthenticator && !(roleManager instanceof CassandraRoleManager))
-            throw new ConfigurationException("CassandraRoleManager must be used with PasswordAuthenticator", false);
+            throw new ConfigurationException(authenticator.getClass().getName() + " requires CassandraRoleManager", false);
```

Comment:
@tiagomlalves you might probably do `CassandraRoleManager.class.getName()` as well. Why we `.getName()` authenticator but not role manager?

Reply:
This was an overlook, thanks for noticing it. I'll fix it.